modules/command-engine/core/src/test/scala/surge/internal/kafka/KafkaProducerActorImplSpec.scala

```diff
@@ -509,7 +509,9 @@ class KafkaProducerActorImplSpec
       probe.watch(fencedOnCommit)
       probe.send(fencedOnCommit, KafkaProducerActorImpl.Publish(testAggs1, testEvents1))
       probe.send(fencedOnCommit, KafkaProducerActorImpl.FlushMessages)
-      probe.expectMsgType[PublishFailure]
+      val result = probe.fishForMessage(10.seconds) { msg => msg.isInstanceOf[PublishResult] }
```

Comment:
I think this can just be the regular `probe.expectMsgType` as it was before since it should always be a `PublishFailure`

Reply:
done